.github/actions/test/action.yml

```diff
@@ -4,4 +4,4 @@ runs:
     using: composite
     steps:
         - shell: bash
-          run: pnpm --workspace-concurrency 1 test:ci
+          run: pnpm test
```

Comment:
`test:ci` was removing all the logs that we don't really care about, are you sure you want to use `test` instead? 🤔

Reply:
`test` leverages turbo,  `test:ci` does not so yes I am sure. What I'm unsure of is how to pass down arguments through turborepo commands

Reply:
found it!, removed test:ci and test:changed